pkg/virt-controller/leaderelectionconfig/config.go

```diff
@@ -39,7 +39,7 @@ func DefaultLeaderElectionConfiguration() Configuration {
 		LeaseDuration: metav1.Duration{Duration: DefaultLeaseDuration},
 		RenewDeadline: metav1.Duration{Duration: DefaultRenewDeadline},
 		RetryPeriod:   metav1.Duration{Duration: DefaultRetryPeriod},
-		ResourceLock:  resourcelock.EndpointsLeasesResourceLock,
+		ResourceLock:  resourcelock.LeasesResourceLock,
```

Comment:
Why?

Reply:
I that const is not public anymore. The const I used `LeasesResourceLock` is not the same though , so I want to see if this option fails..

Reply:
See [here](https://github.com/kubernetes/client-go/blob/35cab326ad02285637a370fcb5a555eb85d1c71a/tools/leaderelection/resourcelock/interface.go#L183)

Reply:
But for some reason the `virt-controller` endpoint id [gone](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/11840/pull-kubevirt-e2e-k8s-1.30-sig-compute/1786017434541494272#1:build-log.txt%3A11560)... trying to figure out why

Reply:
Well evidently, this change made it so that the leader info is kept on the lease. 
Fixed accordingly.
Added a comment on the commit desc.

Reply:
Thanks, makes sense.